### ISSUE TITLE
Create a thumbnail component

### ISSUE THREAD

**Issue by kristoferjoseph** [enhance-dev/wireframe#10]
The thumbnail component consists of an image and optional styling. It accepts user interactions, hover, click, keyboard navigation and selection. Should be considered in drag and drop operations.

![Image](https://github.com/enhance-dev/wireframe/assets/76308/eb596546-a1ca-44a6-9aff-e4c1ec7b2a0a)
